app/queries/intent_metrics_query.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+# Query to return the metrics on the intent selections of the students in a
+# specific draw.
+class IntentMetricsQuery
+  class << self
+    delegate :call, to: :new
+  end
```

Comment:
On second thought, it's nice not to have to call `new`, so I'm going to leave it in and document it.